pyproject.toml

```diff
@@ -24,13 +24,13 @@ classifiers = [
   "Operating System :: OS Independent",
   "Programming Language :: Python",
   "Programming Language :: Python :: 3",
-  "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
   "Topic :: Scientific/Engineering :: Visualization",
 ]
 dependencies = [
-  "pyvista[all]>= 0.44.0",
+  "pyvista[all]>= 0.44.0,<= 0.45.0",
```

Comment:
Is it worth pinning this down in tox rather than here?

Reply:
Yes.